src/material/list/list-item.html

```diff
@@ -1,4 +1,4 @@
-<div class="mat-list-item-content">
+<span class="mat-list-item-content">
   <div class="mat-list-item-ripple" mat-ripple
```

Comment:
_All_ of the divs inside the component would need to become spans.

Reply:
sure, that seems reasonable. I'll change all the div's to spans and make sure they have display: block (or flex)

Reply:
Done.

Note that the `.mat-list-item-content` already has `display: flex` from src/material/list/list.scss, and the `.mat-list-text` already has it's `display` set in _list-common.scss, so we don't need to style those two elements.